contracts/cw-dao-core/Cargo.toml

```diff
@@ -0,0 +1,44 @@
+[package]
+name = "cw-dao-core"
+authors = ["ekez <ekez@withoutdoing.com>"]
+description = "The core module of DAO DAO DAOs."
+license = { workspace = true }
+repository = { workspace = true }
+edition = { workspace = true }
+version = { workspace = true }
```

Comment:
what does `workspace` = true do?

Reply:
this just sets the fields to the same as the workspace `Cargo.toml` file. If we change those values at the root, it'll get propagated everywhere. :) This is nice because all of these fields should be the same for our contracts anyhow (i think).